size_of_longest_substring.cpp

```diff
@@ -0,0 +1,138 @@
+/**
+ * @file
+ * @brief Length of longest substring without repeating characters
+ * @details
+ * From an input string, the length of the longest substring, in which the 
+ * substring is continuous and has no repeating characters, will be the output.
+ * https://leetcode.com/problems/longest-substring-without-repeating-characters/
+ * @author [Rafaela Ferreira](https://github.com/rafaela-eacf)
+ */
+ 
+#include <cassert>
+#include <iostream>
+#include <string>
+#include <algorithm>
+
+/**
+ * @brief Function find_longest_substring from a 
+ * given string returns the length of the longest 
+ * substring without repeating characters
+ * @param string s which will be used to find 
+ * the longest substring 
+ * @returns int value that is equal to the length
+ * of the longest substring
+ */
+int find_longest_substring(std::string s) {
```

Comment:
Consider using `uint64_t` for non-negative values (or their appropriate size: `uint32_t`, `uint16_t`, `uint8_t`) or `int64_t` for negative values. Check other parts of the code ([reference](https://en.cppreference.com/w/cpp/types/integer)). **Let us know if you need any help here or in our [Discord server](https://discord.gg/c7MnfGFGa6).** 🙂